src/pseudopeople/interface.py

```diff
@@ -70,7 +70,10 @@ def _generate_dataset(
         logger.debug(f"Loading data from {data_path}.")
         data = _load_data_from_path(data_path, user_filters)
         if data.empty:
-            continue
+            raise ValueError(
```

Comment:
This will raise if there is no data _in any individual shard_, but we should only raise if there is no data _across all shards_.

Reply:
Yes that's a good point. Im not done with this because I have integration tests failing but thats another thing I need to change.